src/SearchParameters/index.js

```diff
@@ -272,7 +272,7 @@ function SearchParameters(newParameters) {
   /**
    * Enable the ranking informations in the response
    * @see https://www.algolia.com/doc#getRankingInfo
-   * @member {integer}
+   * @member {number}
```

Comment:
REST API says integer so we should keep it like that I guess https://www.algolia.com/doc/rest_api#getRankingInfo